src/lesson6/Animal.java

```diff
@@ -0,0 +1,33 @@
+package lesson6;
+
+public class Animal {
+    private final String name;
+    private static int COUNT_ANIMALS;
```

Comment:
```suggestion
    private static int countAnimals;
```
поскольку данное поле не является константой, оно изменяемо